plugins/base/src/main/kotlin/signatures/JvmSingatureUtils.kt

```diff
@@ -0,0 +1,129 @@
+package org.jetbrains.dokka.base.signatures
+
+import javaslang.Tuple2
+import org.jetbrains.dokka.base.translators.documentables.PageContentBuilder
+import org.jetbrains.dokka.model.*
+import org.jetbrains.dokka.model.properties.WithExtraProperties
+
+interface JvmSingatureUtils {
+
+    fun PageContentBuilder.DocumentableContentBuilder.annotationsBlock(d: Documentable)
+
+    fun PageContentBuilder.DocumentableContentBuilder.annotationsInline(d: Documentable)
+
+    fun <T : Documentable> WithExtraProperties<T>.modifiers(): Set<ExtraModifiers>
+
+    fun Set<ExtraModifiers>.toSignatureString(): String =
+        joinToString("") { it.name.toLowerCase() + " " }
```

Comment:
This appends trailing space, not sure if we want that

Reply:
Actually, it's desired behaviour, in same way previous parts of signature were implemented